django/conf/global_settings.py

```diff
@@ -427,6 +427,8 @@
     'django.middleware.csrf.CsrfViewMiddleware',
 ]
 
+MIDDLEWARES = None
```

Comment:
Ha, no idea :D That might be the reason @carljm called it `MIDDLEWARE` in the first place ;)